lib/runners/processor/cppcheck.rb

```diff
@@ -15,7 +17,8 @@ class NoTargetFiles < SystemError; end
           std: string?,
           project: string?,
           language: string?,
-          'bug-hunting': boolean?
+          'bug-hunting': boolean?,
+          parallel: boolean?
```

Comment:
[nitpick] How about reducing diffs in the future due to a trailing comma?

```suggestion
          parallel: boolean?,
```

Reply:
Fixed. commit ce5fc1446e28541f76c265df2c5f123d6a20efc4